ovos_phal_plugin_ipgeo/__init__.py

```diff
@@ -2,13 +2,15 @@
 from ovos_plugin_manager.phal import PHALPlugin
 from ovos_utils.configuration import get_webcache_location, LocalConf
 from ovos_utils.messagebus import Message
+from ovos_utils.log import LOG
+from urllib3.exceptions import NewConnectionError
 
 
 class IPGeoPlugin(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus, "ovos-phal-plugin-ipgeo", config)
-        self.location = {}
         self.web_config = LocalConf(get_webcache_location())
+        LOG.debug(f'web_config has keys: {self.web_config.keys()}')
```

Comment:
isnt this a bit spammy and excessive? we dont care about this info....

if anything the useful log would be about location being present or not only, thats what we care about, whoever is reading the logs will not know why the keys are logged or what to make of it

also perhaps this should only be logged on actual update?

Reply:
Yes, shouldn't have been committed this was my testing a separate issue